Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -33,6 +33,9 @@ Neos:
         'trustedProxies':
           position: 'start 50'
           middleware: 'Neos\Flow\Http\Middleware\TrustedProxiesMiddleware'
+        'methodOverride':
+          position: 'start 30'
```

Comment:
I put it here instead of after the `parseBody` because we do want the method override to affect the routing (right?) and it previously also only worked with what was available in `$_POST`, i.e. the PHP parsed form/multipart request body. So this is in line with that.